gitRunner.go

```diff
@@ -154,13 +154,18 @@ func gitClone(ctx context.Context, gitName, gitURL, gitBranch string, shallowClo
 }
 
 func gitCheckout(ctx context.Context, gitRevision string) (err error) {
+	args := []string{"fetch", "origin", fmt.Sprintf("HEAD:refs/heads/%s", gitRevision)}
```

Comment:
```suggestion
	args := []string{"fetch", "origin", "--depth", "1", gitRevision}
```

this will allow to fetch commits too